im2txt/README.md

```diff
@@ -0,0 +1,338 @@
+# Show and Tell: A Neural Image Caption Generator
```

Comment:
Is there no way to add this lib as a dependency rather than re-including it? Its not too much of a problem though